pkg/sources/adapter/slacksource/slack.go

```diff
@@ -160,7 +162,7 @@ func (h *slackEventAPIHandler) gracefulShutdown(stopCh <-chan struct{}, done cha
 }
 
 func (h *slackEventAPIHandler) handleError(err error, code int, w http.ResponseWriter) {
-	h.logger.Error("An error ocurred", zap.Error(err))
+	h.logger.Errorw("An error ocurred", zap.Error(err))
```

Comment:
Similarly, using `Errorw()` together with a structured error field seems to ensure the proper escape of newline characters in stringified errors.